bentoml/lightgbm.py

```diff
@@ -204,10 +214,12 @@ def num_replica(self) -> int:
 
     # pylint: disable=arguments-differ,attribute-defined-outside-init
     def _setup(self) -> None:
-        if os.path.splitext(self._model_file)[1] == PKL_EXT:
-            self._model = joblib.load(self._model_file)
-        else:
-            self._model = lgb.Booster(params=self._booster_params, model_file=self._model_file)
+        self._model = load(
+            tag=self._tag,
+            booster_params=self._booster_params,
+            model_store=self._model_store,
+        )
+        self._predict_fn_name = getattr(self._model, self._predict_fn_name)
 
     # pylint: disable=arguments-differ,attribute-defined-outside-init
     def _run_batch(self, input_data: "np.ndarray") -> "np.ndarray":
```

Comment:
instead of `return self._model.predict(...)`, should be `self._predict_fn(...)` instead

Reply:
updated! thank you